app/scripts/modules/core/src/application/search/ApplicationsTable.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { timestamp } from 'core/utils';
+import { IApplicationSummary } from 'core/application';
+
+import { ColHeader } from './ColHeader';
+
+export interface IApplicationTableProps {
+  applications: IApplicationSummary[];
+  currentSort: string;
+  toggleSort: (column: string) => void;
+}
+
+export const ApplicationTable = ({ currentSort, toggleSort, applications }: IApplicationTableProps) => (
+  <table className="table table-hover">
+    <thead>
+    <tr>
```

Comment:
not indented

Reply:
Webstorm thinks this shouldn't be indented.  I removed these thead/tfoot/tbody from the list now.

![screen shot 2018-02-02 at 11 10 11 am](https://user-images.githubusercontent.com/2053478/35750418-c37bab24-0809-11e8-8ac9-d4704992acff.png)

Reply:
what's your Dart config look like?

Reply:
I tried Dart for a while and sorta liked it